### PR TITLE
Desktop feature

### PR DESCRIPTION
**Pull request by SabiMantock**
Desktop main features:

- Added background images for desktop view
- Added styling for desktop view using media queries to give it some responsiveness
- Added nav links to index.html

Portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,195 +126,208 @@ <h3 id="title">Tonic</h3>
...
+              <li class="lang">
+                <div class="langImg">
+                  <img src="images//html.png" alt="" />
+                </div>
+                <h4>HTML</h4>
+              </li>
+              <li class="lang">
+                <div class="langImg">
+                  <img src="images/css.png" alt="" />
+                </div>
```
Comment (Olamarx) [inline index.html:315]: - [ ] These images should be in rows not columns, kindly take a second look at it.
### Your Work
image
### Figma
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @SabiMantock, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed 👍🏼
- Meaningful commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._
### I would have made this comment inline but I could not find the code. It has been merged as this is just a modification for desktop version.
- [x] Kindly take note of the position of the content of the name, paragraph and lets connect and the icons. They are too close to the navigation and leave too much space underneath it.
### Figma
image

### Your design
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (SabiMantock) [reply index.html:315]: thank you for the review, would make the required changes

**Review by christianbiring1 — APPROVED** [review]
Hi @SabiMantock , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commits messages :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**